js/chatbot/shared/Message.svelte

```diff
@@ -249,6 +249,11 @@
 	.message {
 		position: relative;
 		width: 100%;
+		margin-bottom: var(--spacing-md);
```

Comment:
Just looking at the story book, I feel like it increases the spacing for non-consecutive messages a lot

<img width="1274" alt="image" src="https://github.com/user-attachments/assets/08df7bd0-aab6-4a69-9cb9-a51dda137de4" />

Reply:
Ok let me try something. The issue with the original behavior happens when you have multiple messages from the same role -- the bubbles have no gap, which looks weird